mpv.el

```diff
@@ -362,5 +362,14 @@ of \\[universal-argument] will add another `mpv-seek-step' seconds."
   (interactive)
   (mpv--enqueue '("revert-seek") #'ignore))
 
+;;;###autoload
+(defun mpv-version ()
+  "Return mpv version."
+  (interactive)
+  (let ((version (cadr (split-string (car (process-lines "mpv" "--version"))))))
+    (prog1 version
+      (if (called-interactively-p 'interactive)
+	  (message "%s" version)))))
```

Comment:
Generally the `…-version` interactive commands in Emacs seem to also display the program name: 
```suggestion
	  (message "mpv %s" version)))))
```
We _could_ also print the full first line of the `--version` output, or even the complete output? (Only during interactive use, the function would still only return the "reduced" version.)